packages/brain/src/modules/cron/index.ts

```diff
@@ -63,39 +63,50 @@ export class Cron {
     try {
       logger.debug(`Reloading data...`);
 
-      // 0. GET data
+      // 0. GET status
+      const signerApiStatus = await this.signerApi.getStatus();
+
+      // If web3signer API is not UP, skip data reload and further steps.
+      // This is done to avoid unintended DB modifications when the API is down.
+      // Status can be "UP" | "DOWN" | "UNKNOWN" | "LOADING" | "ERROR";
+      if (signerApiStatus.status !== "UP") {
```

Comment:
could you show any error message returned from the websigner for more context

Reply:
what do you mean? if `getStatus()` throws an error, it will be logged since the call is inside a trycatch block. 

I have updated the warning message so it shows the value of `signerApiStatus.status` if it's not up